src/Microsoft.AspNetCore.SignalR.Common/CloseMessage.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.AspNetCore.SignalR.Internal.Protocol
+namespace Microsoft.AspNetCore.SignalR
```

Comment:
Hmmm, I'm not sure about putting them in the main namespace. How often will devs interact with these classes?

Reply:
You think a .Protocol namespace?

Reply:
Yeah

Reply:
Will do.